sql/postgresql/pggrammar.g4

```diff
@@ -0,0 +1,4294 @@
+parser grammar pggrammar;
```

Comment:
It's better to rename to `PostgreSQLParser`. We use `Lexer` and `Parser` suffixes in all grammars in this repository.

```suggestion
parser grammar PostgreSQLParser;
```